### PR TITLE
docs: add links with anchors

### LINKED ISSUE #439: Github main readme use the anchor `.../cli#readme` instead of `.../cli`

### PR DESCRIPTION
**Pull request by BioPhoton**
adresses parts of #370 
closes #439

### CODE REVIEW

File: packages/cli/docs/images/cli-run-stdout-example.png
Comment (matejchalk) [inline packages/cli/docs/images/cli-run-stdout-example.png:1]: Was this run via `nx`? The reason I ask is that we know it omits separator lines, so this report looks pretty bunched up. If run the CLI directly (`node ./dist/...`), then it should look better.

File: packages/cli/README.md [hunk 1]
```diff
@@ -67,7 +71,15 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
    };
    ```
 
-4. Define your custom categories.
+4. Run the CLI with `npx code-pushup` (see `--help` for list of commands and arguments).
+
+5. View report file(s) in output directory (specified by `persist.outputDir` configuration).
+
+<img src="./docs/images/cli-run-stdout-example.png" width="400" alt="example of code-pushup terminal output">
```
Comment (matejchalk) [inline packages/cli/README.md:78]: Not sure I'd include this screenshot here, then contents are misleading (no categories or example plugins mentioned in above setup).

File: packages/cli/README.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
 ## Getting started
 
 1. Install as a dev dependency with your package manager:
+   <details>
+   <summary>Install options for <code>npm</code>, <code>yarn</code>,<code>pnpm</code> etc</summary>
```
Comment (matejchalk) [inline packages/cli/README.md:27]:
```suggestion
   <summary>Installation command for <code>npm</code>, <code>yarn</code> and <code>pnpm</code></summary>
```

File: packages/cli/README.md (hunk 1 above)
Comment (BioPhoton) [reply packages/cli/README.md]: removed

File: packages/cli/docs/images/cli-run-stdout-example.png
Comment (BioPhoton) [reply packages/cli/docs/images/cli-run-stdout-example.png:1]: Yes I ran it with nx.

Comment (BioPhoton) [reply packages/cli/docs/images/cli-run-stdout-example.png:1]: I will run it again directly and make a new screenshot

Comment (BioPhoton) [reply packages/cli/docs/images/cli-run-stdout-example.png:1]: added new screen shot.

File: packages/cli/README.md [hunk 3]
```diff
@@ -67,7 +71,13 @@ _If you're looking for programmatic usage, then refer to the underlying [@code-p
    };
    ```
 
-4. Define your custom categories.
+4. Run the CLI with `npx code-pushup` (see `--help` for list of commands and arguments).
+
+5. View report file(s) in output directory (specified by `persist.outputDir` configuration).
+
+### Setup categories
```
Comment (Tlacenka) [inline packages/cli/README.md:78]:
```suggestion
### Set up categories (optional)
```

File: packages/cli/docs/images/cli-run-stdout-example.png
Comment (Tlacenka) [inline packages/cli/docs/images/cli-run-stdout-example.png:1]: The used screenshot has a bit of a broken layout - all values except one span multiple lines. Could you use audit reports with values that fit in one line?

Comment (matejchalk) [reply packages/cli/docs/images/cli-run-stdout-example.png:1]: Maybe you could "cheat" and increase the column width just to get a nicer screenshot :grin:

**Review by matejchalk — APPROVED** [review]

**Review by matejchalk — APPROVED** [review]

**Review by Tlacenka — APPROVED** [review]

**Pull request merged**